packages/server/src/koaJsonRpc/index.ts

```diff
@@ -99,18 +100,21 @@ export default class KoaJsonRpc {
         ctx.request.method !== 'POST'
       ) {
         ctx.body = jsonResp(body.id || null, new InvalidRequest(), undefined);
+        ctx.status = 400;
```

Comment:
does it make sense to add the status as an optional(?) forth parameter to the jsonResp method?

Reply:
I'm not sure I understand

Reply:
Ah, I see my suggestion does not make sense.  My mistake.  

But I do have another question.  What happens when calling jsonResp and it throws an exception?

Reply:
Hm, good question. It seems like it's not being handled by us anywhere. cc: @georgi-l95

Reply:
```
curl localhost:7546 -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"nonExistingMethod","params": [],"id":1}'
Internal Server Error%                                                                  
```

Reply:
sounds reasonable